docker-compose.yml

```diff
@@ -1,12 +1,25 @@
 version: "3"
 services:
   hapi-fhir-jpaserver-start:
+    network_mode: host
```

Comment:
You might not want to include this since it doesn't work on all platforms. It causes problems when I try it on a Mac.